app/assets/js/admin-upload-terms.js

```diff
@@ -10,69 +10,110 @@
                     error_msg += rtm_upload_terms_error_msgs.error_msg;
+                    var general_upload_terms_error_message_parents_el = general_upload_terms_error_message.parents( '.rtm-content' ).attr( 'id' );
+
+                    if ( 0 < general_upload_terms_error_message_parents_el.length ) {
+                        // navigate the admin to this location, so he can know about this error.
+                        $( '#tab-' + general_upload_terms_error_message_parents_el ).click();
+                    }
+
                     return rtp_show_error_message ( general_upload_terms_error_message, error_msg );
                 }
             }
         }
         if ( return_code && general_upload_terms_show_pricacy_message.length > 0 && 'undefined' !== typeof general_upload_terms_show_pricacy_message ) {
             var error_msg = "";
+
             if ( general_upload_terms_show_pricacy_message.prop( 'checked' ) ) {
-                jQuery( '.error_msg' ).remove();
+                $( '.error_msg' ).remove();
 
                 /* Check "Terms of Service Message" Emply Or Not */
-                if ( general_upload_terms_privacy_message.val().trim() == '' ) {
+
+                if ( '' === general_upload_terms_privacy_message.val().trim() ) {
                     error_msg += rtm_upload_terms_error_msgs.privacy_msg;
+                    var general_upload_terms_privacy_message_parents_el = general_upload_terms_privacy_message.parents( '.rtm-content' ).attr( 'id' );
+
+                    if ( 0 < general_upload_terms_privacy_message_parents_el.length ) {
+                        // navigate the admin to this location, so he can know about this error.
+                        $( '#tab-' + general_upload_terms_privacy_message_parents_el ).click();
+                    }
+
                     return rtp_show_error_message ( general_upload_terms_privacy_message, error_msg );
                 }
             }
         }
     } );
 
-    /* Show Error Message If Incorrect Validation  */
+    /**
+	 * This appends the error message to the received selector, showing that the validation has failed.
+	 *
+	 * @param selector
+	 * @param error_msg
+	 * @returns {boolean}
+	 */
     function rtp_show_error_message( selector, error_msg ) {
-        var elm_selector = jQuery( selector );
+        var elm_selector = $( selector );
         elm_selector.focus();
         elm_selector.css( 'border-color', 'red' );
-        if ( elm_selector.parent().length > 0 && 'error_msg' !== elm_selector.parent().attr( 'class' ) ) {
-            var invalid_error_msg = jQuery( "<span />" ).attr( 'style', 'display:block' ).addClass( 'error_msg' ).html( error_msg );
+        var elm_selector_parent = elm_selector.parent();
+
+        if ( elm_selector_parent.length > 0 && 'error_msg' !== elm_selector_parent.attr( 'class' ) ) {
+            var invalid_error_msg = $( "<span />" ).attr( 'style', 'display:block' ).addClass( 'error_msg' ).html( error_msg );
```

Comment:
Use single quote and use .css() instead of att() with styles.
Ref.: https://api.jquery.com/css/
```suggestion
            var invalid_error_msg = $( '<span />' ).css( 'display', 'block' ).addClass( 'error_msg' ).html( error_msg );
```